README.md

```diff
@@ -1,147 +1,101 @@
+                sh 'infracost breakdown --path plan.json --format json --out-file infracost.json'
 
-Terragrunt users should also read [this page](https://www.infracost.io/docs/iac_tools/terragrunt). Terraform Cloud/Enterprise users should also read [this page](https://www.infracost.io/docs/iac_tools/terraform_cloud_enterprise).
+                // IMPORTANT: update this depending on which VCS provider you use and which plugin you are using.
+                // Infracost comment supports GitHub, GitLab, Azure Repos and Bitbucket
+                // You will need to update the environment variables below to match your environment.
+                // For the full list of options, see: https://www.infracost.io/docs/features/cli_commands/#comment-on-pull-requests
+                sh 'infracost comment github --path infracost.json --repo $GITHUB_REPO --pull-request $GITHUB_PULL_REQUEST_NUMBER --github-token $GITHUB_TOKEN'
+            }
+        }
+    }
+}
+```
 
-### `INFRACOST_API_KEY`
+3. 🎉 That's it! Send a new pull request to change something in Terraform that costs money. You should see a merge request comment that gets updated, e.g. the 📉 and 📈 emojis will update as changes are pushed! Check the build Console Output and [this page](https://www.infracost.io/docs/troubleshooting/) if there are issues.
 
-**Required** To get an API key [download Infracost](https://www.infracost.io/docs/#quick-start) and run `infracost register`.
+## Comment options
 
-### Cloud credentials
+For different commenting options `infracost comment` command supports the following flags:
 
-**Required** You do not need to set cloud credentials if you use Terraform Cloud/Enterprise's remote execution mode, instead you should follow [this page](https://www.infracost.io/docs/iac_tools/terraform_cloud_enterprise).
+- `--behavior <value>`: Optional, defaults to `update`. The behavior to use when posting cost estimate comments. Must be one of the following:
+  - `update`: Create a single comment and update it on changes. This is the "quietest" option. Pull request followers will only be notified on the comment create (not updates), and the comment will stay at the same location in the comment history.
+  - `delete-and-new`: Delete previous cost estimate comments and create a new one. Pull request followers will be notified on each comment.
+  - `hide-and-new`: Minimize previous cost estimate comments and create a new one. Pull request followers will be notified on each comment. Only supported by GitHub.
+  - `new`: Create a new cost estimate comment. Pull request followers will be notified on each comment.
+- `--pull-request <pull-request-number>`: Required when posting a comment on a pull request. Mutually exclusive with `--commit` flag.
+- `--commit <commit-sha>`: Required when posting a comment on a commit. Mutually exclusive with `--pull-request` flag. Not available when bitbucket-server-url is set.
```

Comment:
As we don't specify the subcommand here I think we don't need to mention Bitbucket. What do you think?

```suggestion
- `--commit <commit-sha>`: Required when posting a comment on a commit. Mutually exclusive with `--pull-request` flag.
```